src/test/java/uk/gov/hmcts/sptribs/common/notification/CaseUnlinkedNotificationTest.java

```diff
@@ -38,7 +38,7 @@ void shouldNotifySubjectOfApplicationReceivedWithEmail() {
         //Given
```

Comment:
Think these test names need updating since they do not notify that the application has been received.